service/kvmclusterconfig/v1/key/error.go

```diff
@@ -8,3 +8,10 @@ var wrongTypeError = microerror.New("wrong type")
 func IsWrongTypeError(err error) bool {
 	return microerror.Cause(err) == wrongTypeError
 }
+
+var emptyValueError = microerror.New("empty value")
```

Comment:
Lets keep the order alphabetical please.

Reply:
Sure. I'll fix this. Trying to remember but constantly forgetting...